docs/types-and-platforms/server-types/hybrids.md

```diff
@@ -8,14 +8,24 @@ A [Magma](https://magmafoundation.org/) server, which is a combination of Forge
 
 !!! note
 
-The Magma project [has benn terminated](https://git.magmafoundation.org/magmafoundation/magma-1-20-x/-/commit/4e7abe37403c47d09b74b77bcfc26a19b18f5891). Please use Ketting for 1.20.2+.
+The Magma project [has been terminated](https://git.magmafoundation.org/magmafoundation/magma-1-20-x/-/commit/4e7abe37403c47d09b74b77bcfc26a19b18f5891). Please use Magma Maintained for 1.12.2, 1.18.2, 1.19.3, and 1.20.1, or Ketting for 1.20.2+.
 
 There are limited base versions supported, so you will also need to  set `VERSION`, such as "1.12.2", "1.16.5", etc.
 
+### Magma Maintained
+A [Magma Maintained](https://github.com/magmamaintained/) server, which is a alternative project of Magma, can be used with
+
+    -e TYPE=MAGMA_MAINTAINED
+
+!!! note
+
+There are limited base versions supported, so you will also need to set `VERSION`, such as "1.12.2", "1.18.2", "1.19.3", or "1.20.1".

```

Comment:
This needs to be indented with four spaces to be rendered within the note.

Can view docs rendering at https://docker-minecraft-server--2552.org.readthedocs.build/en/2552/types-and-platforms/server-types/hybrids/#magma-maintained

Reply:
Thanks for finding that.

I have fixed that including other sections in this file.

Reply:
> Thanks for finding that.
> 
> 
> 
> I have fixed that including other sections in this file.

Thanks. Yes, I have been gradually finding other places where I missed the indent.